lib/open_api_spex/open_api/map.ex

```diff
@@ -0,0 +1,395 @@
+defmodule OpenApiSpex.OpenApi.Map do
```

Comment:
`OpenApiSpex.OpenApi.Decoder` feels like it would communicate the intention of this module slightly better.

Reply:
Addressed.